backend/internal/repository/get_image.go

```diff
@@ -1,36 +1,30 @@
 package repository
 
 import (
+	"fmt"
 	"log"
 
-	"math/rand"
-	"time"
-
 	"github.com/hoka-isdl/MERS/backend/internal/schema"
 )
 
-func GetImages() []schema.ListImagesInner {
+func RandGetImages(images_num int) ([]schema.ListImagesInner, error) {
 	var images schema.ListImagesInner
```

Comment:
細かいけど、ここは複数形じゃなくて単数形の方が実際の処理に則してて良いと思う。他の箇所も一括置換とかで修正しておいてほしい。
```suggestion
var image schema.ListImagesInner
```

Reply:
imagesをimageに変更しました．
ご確認お願いいたします．

commit url: [abcc5d0](https://github.com/hoka-isdl/MERS/pull/19/commits/abcc5d08ada43e4dcc80ff09c3bb086a85d64094)